### ISSUE TITLE
Seems to have stopped working altogether - Firefox 132.0.2

### ISSUE THREAD

**Issue by GanizaniSitara** [esteinmann/chatgpt-convdown#57]
Very useful extension, I don't know exactly when over last couple of weeks, but seems no longer working on 132.0.2

**Comment by esteinmann** [comment]
Is this still the case? I cannot reproduce with Firefox 133.0

**Comment by GanizaniSitara** [comment]
No, seems to have corrected itself, but can't find the root cause.
